src/staking/index.ts

```diff
@@ -444,4 +254,93 @@ export class Staking {
       );
     }
   }
+
+  /**
+   * Create a slash timelock unbonded transaction for staking.
+   * 
+   * @param {Transaction} stakingTx - The staking transaction to slash.
+   * without coordinates.
+   * 
+   * @returns {PsbtTransactionResult} - An object containing the unsigned psbt and fee
+   * 
+   * @throws {StakingError} - If the delegation is invalid or the transaction cannot be built
+   */
+  public createSlashTimelockUnbondedTransaction(
```

Comment:
I would say this this naming (as well as ` createSlashEarlyUnbondedTransaction`)  is a bit unclear.

We have two slashing transactions in the system:
- staking slashing transaction - i.e slashing transaction which spent staking output
- unbonding slashing transactins - i.e slashing transactions which spend unbonding output

so maybe `createStakingOutputSlashingTransaction` and `createUnbondingOutputSlashingTransaction` ?